### PR TITLE
Rename member variables in BufferTypeBase:

### PR DESCRIPTION
**Pull request by toregge**
minArrays             => min_entries
maxArrays             => max_entries
numArraysForNewBuffer => num_entries_for_new_buffer

@vekterli : please review
@geirst : FYI

### CODE REVIEW

**Review by vekterli — DISMISSED** [review]
:+1:

**Review by vekterli — APPROVED** [review]

File: vespalib/src/vespa/vespalib/datastore/buffer_type.h [hunk 1]
```diff
@@ -114,10 +114,10 @@ class BufferTypeBase
     };
 
     uint32_t _arraySize;  // Number of elements in an allocation unit
-    uint32_t _minArrays;  // Minimum number of arrays to allocate in a buffer
-    uint32_t _maxArrays;  // Maximum number of arrays to allocate in a buffer
+    uint32_t _min_entries;  // Minimum number of arrays to allocate in a buffer
```
Comment (geirst) [inline vespalib/src/vespa/vespalib/datastore/buffer_type.h:117]: @toregge : Please update comment

File: vespalib/src/vespa/vespalib/datastore/buffer_type.h [hunk 2]
```diff
@@ -114,10 +114,10 @@ class BufferTypeBase
...
+    uint32_t _max_entries;  // Maximum number of arrays to allocate in a buffer
```
Comment (geirst) [inline vespalib/src/vespa/vespalib/datastore/buffer_type.h:118]: @toregge : Please update comment

**Review by geirst — COMMENTED** [review]
👍

File: vespalib/src/vespa/vespalib/datastore/buffer_type.h (hunk 1 above)
Comment (toregge) [reply vespalib/src/vespa/vespalib/datastore/buffer_type.h:117]: Done.

File: vespalib/src/vespa/vespalib/datastore/buffer_type.h (hunk 2 above)
Comment (toregge) [reply vespalib/src/vespa/vespalib/datastore/buffer_type.h:118]: Done

**Pull request merged**